src/constants/coreSettings.ts

```diff
@@ -410,18 +410,11 @@ export const CORE_SETTINGS: SettingParams[] = [
     versionAdded: '1.3.5'
   },
   {
-    id: 'Comfy.Keybinding.UnsetBindings',
```

Comment:
Why do we need to nuke old settings. Doing so would break all existing users' keybinding setup.

Reply:
Good point. When I have more time, I can build a migration function